packages/zendesk-support-adapter/src/auth.ts

```diff
@@ -49,7 +49,7 @@ export const usernamePasswordCredentialsType = createMatchingObjectType<
     },
     subdomain: {
       refType: BuiltinTypes.STRING,
-      annotations: { _required: true },
+      annotations: { _required: true, message: 'subdomain (https://<your subdomain>.zendesk.com)' },
```

Comment:
1. nit: please separate the lines
```suggestion
      annotations: {
        _required: true,
        message: 'subdomain (https://<your subdomain>.zendesk.com)',
      },
```
2. I think we should add the same below for the oauth parameters below in this file (maybe we should extract the message to a shared string). lmk if you prefer that I do it

Reply:
Thanks @netama fixed now